crates/ide_assists/src/handlers/replace_derive_with_manual_impl.rs

```diff
@@ -2,6 +2,8 @@ use hir::ModuleDef;
 use ide_db::helpers::{import_assets::NameToImport, mod_path_to_ast};
 use ide_db::items_locator;
 use itertools::Itertools;
+use syntax::ast::edit::AstNodeEdit;
+use syntax::ted;
```

Comment:
```suggestion
use ide_db::{helpers::{import_assets::NameToImport, mod_path_to_ast}, items_locator};
use itertools::Itertools;
use syntax::{ast::edit::AstNodeEdit, ted};
```
auto import used the wrong merging strategy here since the file was using the wrong one already thanks to the ide_db imports

Reply:
ah, good point. Fixed!